cypress/runLocal.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+export CYPRESS_baseUrl=${CYPRESS_baseUrl:-https://nextcloud.local/index.php}
```

Comment:
This is not applicable for me. But I guess I will have to modify it locally?

Reply:
Exactly. You can just run it with `CYPRESS_baseUrl=http://whatever.ncdev/index.php ./runLocal.sh open`.